cluster-support-bot.py

```diff
@@ -1,13 +1,28 @@
 import argparse
 import logging
 import os
+import re
 import time
 
-import hydra
+import prometheus_client
 import slack
+
+import hydra
 import telemetry
 
 
+mention_counter = prometheus_client.Counter('proactive_support_mentions',
+        'Number of times a cluster is mentioned in #proactive-support', ['_id'])
+comment_counter = prometheus_client.Counter('proactive_support_comments',
+        'Number of times a cluster has been commented #proactive-support', ['_id'])
+set_summary_counter = prometheus_client.Counter('proactive_support_set_summary',
+        'Number of times a cluster summary has been set #proactive-support', ['_id'])
+# Eventually we'll likely switch to some sort of wsgi app but for now any path
+# requested will return our metrics.  We'll configure /metrics to be scrapped
+# so we can leave room for some sort of landing page in the future.
+prometheus_client.start_http_server(8080)
```

Comment:
This may also allow us to set [health checks][1] on our Deployment.  Do we want to document that in `install.md`?  I'm also fine punting that to future work.

[1]: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/

Reply:
yeah, let's do that in a future commit.  We definitely should have a health check!